plane/source/docs/apms-failsafe-function.rst

```diff
@@ -7,111 +7,85 @@ Plane Failsafe Function
 Plane has a limited failsafe function which is designed to do four
 things:
 
-#. Detect a complete loss of RC signal and initiate a defined response, such as returning to home. Detection is either by lack of data/pulses from the receiver, the throttle channel PWM value falling below a certain point set by :ref:`THR_FS_VALUE<THR_FS_VALUE>`, or the receiver sets a FS bit in its data stream for those protocols supporting this. Detection of these must be enabled by setting :ref:`THR_FAILSAFE<THR_FAILSAFE>` = 1 or 2.
+#. Detects a RC Failsafe condition and then initiating a defined response, such as returning to home. Detection of an RC Failsafe is either a complete loss, or corruption, of RC signals, or the receiver sets a FS bit in its data stream for those protocols supporting it (SBUS, etc.), or that the throttle channel PWM value falls below a certain point set by :ref:`THR_FS_VALUE<THR_FS_VALUE>`. This RC failsafe must be enabled by enabled by by setting :ref:`THR_FAILSAFE<THR_FAILSAFE>` = 1.
```

Comment:
```suggestion
#. Detects a RC Failsafe condition and then initiating a defined response, such as returning to home. Detection of an RC Failsafe is either a complete loss, or corruption, of RC signals, or the receiver sets a FS bit in its data stream for those protocols supporting it (SBUS, etc.), or that the throttle channel PWM value falls below a certain point set by :ref:`THR_FS_VALUE<THR_FS_VALUE>`. This RC failsafe must be enabled by setting :ref:`THR_FAILSAFE<THR_FAILSAFE>` = 1.
```

Reply:
should maybe note in here somewhere that RC_OPTIONS bit 2 if set will not allow a failsafe to be detected on the protocols that support it.